pydocstringformatter/configuration/arguments_manager.py

```diff
@@ -69,6 +69,16 @@ def _register_arguments(self, version: str) -> None:
             ),
         )
 
+        self.configuration_group.add_argument(
+            "--push-multiline-opening",
```

Comment:
This option name is hard to understand I had to read the description and the change in the functional tests.  I don't have a very clear option name to propose though. Maybe

```suggestion
            "--first-line-summary",
```


```suggestion
            "--multilines-with-summary",
```
?

Reply:
Hmm, agreed although those names also don't really fit the function I think.

It should be something with `multiline`, `quotes/position` and perhaps `summary`.

Reply:
``--summary-next-to-quote-for-multiline`` ? it's a little long but it could work.

Reply:
I went with ``--summary-quotes-same-line``. Would that be okay for you?

Reply:
Sounds fine, the long description will help anyway :)